Viewer/src/managers/observablesManager.ts

```diff
@@ -54,6 +54,15 @@ export class ObservablesManager {
      */
     public onFrameRenderedObservable: Observable<any>;
 
+    /**
+     * Will notify when VR mode is entered.
+     */
+    public onEnteringVRObservable: Observable<any>;

```

Comment:
Can you confirm that this should be public (eg. external developers will use this), or would it make sense to be private and only used internally by the viewer?

Reply:
These observables are public because the SceneManager and Viewer both need to be able to access them.